skops/card/_model_card.py

```diff
@@ -1410,13 +1425,18 @@ def save(self, path: str | Path) -> None:
         path: str, or Path
             Filepath to save your card.
 
+        plot_path: str
+            Filepath to save the plots.
```

Comment:
Could you please elaborate on this? E.g. mention under what circumstances users may want to use this parameter.